official/bert/bert_models.py

```diff
@@ -192,13 +201,14 @@ def call(self, inputs):
     denominator = tf.keras.backend.sum(lm_label_weights) + 1e-5
     mask_label_loss = numerator / denominator
 
-    sentence_labels = tf.keras.backend.cast(unpacked_inputs[4], dtype=tf.int32)
+    sentence_labels = unpacked_inputs[4]
     sentence_labels = tf.keras.backend.reshape(sentence_labels, [-1])
     sentence_label_one_hot = tf.keras.backend.one_hot(sentence_labels, 2)
     per_example_loss_sentence = -tf.keras.backend.sum(
         sentence_label_one_hot * sentence_output, axis=-1)
     sentence_loss = tf.keras.backend.mean(per_example_loss_sentence)
     loss = mask_label_loss + sentence_loss
+    # TODO(hongkuny): Avoids the hack and switches add_loss.
```

Comment:
nit: `Avoid the hack and switch to using add_loss() instead.'

Reply:
will do in g3. thx